docs/writing-tests/making-a-testing-plan.md

```diff
@@ -281,22 +233,22 @@ confer with the people who are maintaining the tests.
 
 ### Avoid excessive breadth
 
-Many algorithms operate on input that takes the form of a scalar value. When
-the set of values is finite, it can be tempting to test them all exhaustively.
-When the set is very large, test authors can reduce repetition by defining
-tests programmatically in loops.
+When the set of input values is finite, it can be tempting to test them all
+exhaustively. When the set is very large, test authors can reduce repetition by
+defining tests programmatically in loops.
 
 Using advanced control flow techniques to dynamically generate tests has a
 number of drawbacks. It makes the test suite more difficult to understand since
 readers have to mentally "unwind" the iteration to determine what is actually
 being verified. It also increases the risk of bugs in the tests. These bugs may
 not be obvious--they may not cause failures, and they may exercise fewer cases
-than intended.
+than intended. Finally, tests authored using this approach often have execution
```

Comment:
I understand this because of the context of our discussion, but I think saying just that such tests can easily take too long to run would be clearer.